src/rt/sched/cown.h

```diff
@@ -115,6 +115,8 @@ namespace verona::rt
 
     std::atomic<Status> status{};
     std::atomic<Priority> bp_state = Priority::Normal;
+    // TODO: combine into `bp_state` field
+    Cown* blocker = nullptr;
```

Comment:
I think this should be `std::atomic` as it is a communication channel between threads.